libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -180,8 +180,10 @@ class YulUtilFunctions
 	std::string storageArrayPopFunction(ArrayType const& _type);
 
 	/// @returns the name of a function that pushes an element to a storage array
+	/// _fromType represents type of value passed to the function (type of element being pushed)
+	/// if _fromType is different than _type.baseType() there conversion will happen (in case of reference type deep copy will happen)
```

Comment:
"there conversion will happen"? What do you mean here?

Reply:
Probably should remove that part, as it actually only do deep copy, so I should state that